### DISCUSSION TITLE
Failed to load resource: the server responded with a status of 500 (Internal Server Error)

### DISCUSSION THREAD

**Discussion by miczlo** [blakeblackshear/frigate#14174] [General Support] [answered]
Frigate was working fine for ages, did a host reboot and now it's only rendering menus and not cameras, configs etc.
Tried Chrome and Firefox
Tried updating from stable to 14.1
Any ideas where to look?
Edit: I get same thing via proxy and via IP:5000, I tried curling from within the host to localhost:5000/profile and same Error 500
![image](https://github.com/user-attachments/assets/38410683-1189-408a-b1ad-ac6129c1e2a8)

**Comment by hawkeye217** [comment]
We will need to see your frigate logs and config in order to be able to help.

**Reply by miczlo** [reply]
Yeah fair enough
Just to add, I get same thing via proxy and via IP:5000, I tried curling from within the host to localhost:5000/profile and same Error 500
Logs:
Edited, Left the relevant bit after finding the solution
 ```
frigate  | 2024-10-06 13:05:37.845869033  Address already in use
frigate  | 2024-10-06 13:05:37.845893865  Port 5001 is in use by another program. Either identify and stop that program, or start the server with a different port.
```

Config:
```
Removed as was not relevant

```

**Comment by hawkeye217** [comment, marked as answer]
Are you running via docker? If so, what is your docker compose?

There are some problematic errors in the logs:

```
frigate  | 2024-10-06 13:05:37.845869033  Address already in use
frigate  | 2024-10-06 13:05:37.845893865  Port 5001 is in use by another program. Either identify and stop that program, or start the server with a different port.
```

Also, Frigate can't connect to `back_tree`, `garden_tapo`, and `garden`. The logs are showing network errors.

**Reply by miczlo** [reply]
I don't know how I missed the 5001. Was this port always used by frigate? I had a small self put app for monitoring frigate that was running on 5001 but it would have been there for years.
Anyway it's fine after stopping the other 5001 app - thanks for helping me read the logs, I'm clearly blind

**Reply by hawkeye217** [reply]
Port usage just depends how you've configured Frigate to run. 

No problem, happy to help.
